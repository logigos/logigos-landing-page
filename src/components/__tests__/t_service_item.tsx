import React from 'react'
import renderer from 'react-test-renderer'
import img_sender from '../../images/services-section/image/sender.svg'
import ServiceItem from '../common/ServiceFeature'

describe('ServiceItem', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(
                <ServiceItem
                    image={img_sender}
                    name={'test_name'}
                    detail={'test_detail'}
                />
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})
