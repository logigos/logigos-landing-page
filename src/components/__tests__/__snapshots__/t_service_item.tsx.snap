// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceItem renders correctly 1`] = `
<div
  className="sc-AxjAm sc-AxhUy sc-AxgMl sc-Axmtr fAqYOc"
>
  <img
    className="sc-AxmLO fwvVin"
    name="test_name"
    src="test-file-stub"
  />
  <p
    className="sc-fzozJi bcXsGH"
  >
    test_name
  </p>
  <p
    className="sc-fzoLsD dGGtn"
  >
    test_detail
  </p>
</div>
`;
